src/main/java/tetris/Tetromino.java

```diff
@@ -139,63 +173,42 @@ private void createUShape() {
     }
 
     private void createMiniShape() {
-        int x = Grid.WIDTH/2;
+        int x = Grid.WIDTH / 2;
         int y = Grid.HEIGHT;
         for (int i = 0; i < 2; i++) {
             miniShape.add(new Square(x + i, y));
         }
     }
 
-    public List<Square> getOShape() {
```

Comment:
i deleted all these bc i don't think other classes will need to access each shape, just the current one but let me know otherwise